src/statistics.cpp

```diff
@@ -208,6 +230,14 @@ static void merge_battle_result_maps(stats::battle_result_map& a, const stats::b
 	}
 }
 
+static void merge_cth_map(std::map<int, struct statistics::stats::by_cth_t>& a, const std::map<int, struct statistics::stats::by_cth_t>& b)
```

Comment:
I've defined a type alias using `typedef` to be consistent with other member types of `statistics::stats` and used it throughout, which addressed most of your comments about removing the `struct` keyword.

Reply:
For the record, `typedef` is more common in existing code because the using aliases were only introduced in C++11. However, I'm also fine with `typedef`.